bark/runtime/runtime.py

```diff
@@ -53,10 +53,11 @@ def step(self):
     if len(self._world_history) == 0 and self._maintain_world_history:
         self._world_history.append(self._world.Copy())
 
-    self._world.Step(self._step_time)
+    self._world.PlanAgents(self._step_time)
 
     if self._render:
         self.render()
+    self._world.Execute(self._step_time)
```

Comment:
That means that render always plots the old world state.

Reply:
Yes, thats a good idea:
- It allows to draw planned mcts states or other information from the planner, ontop of the world state it was planned for
- It allows to draw the correct senses world ontop of the true world

Reply:
As first step in BARK we call runtime.reset() which returns `s_0`.

Then we render the world after `PlanAgents` which still returns `s_0`.

So de facto, `s0` is then rendered twice. Is this intended?

Reply:
I agree with Julian

Reply:
We have to find a way to draw the last state on reset()